### ISSUE TITLE
reassess tutorial tests min / max accuracies and make tutorial tests less flaky

### ISSUE THREAD

**Issue by npapernot** [tensorflow/cleverhans#176]

**Comment by npapernot** [comment]
see #170

**Comment by abhibhav14** [comment]
I am currently working on it right now, For the MNIST tutorials, we can increase the adversarial examples after adversarial training from 0.07 to 0.70.
However, I just realized that switching to Adam messed up the blackbox tutorial. Originally, test accuracy of oracle on samples generated by substitute was roughly 0.5, but switching to Adam has actually made that worse, it is now a little over 0.8.

**Comment by npapernot** [comment]
@abhibhav14  Did you make any progress on this? 

If not, I'll probably make a PR soon to address this issue because some of the tests are more frequently flaky because of the new optimizers

**Comment by abhibhav14** [comment]
For each of the tests, I have reassessed the accuracies and made a pull request at #197 . For the blackbox attack, the accuracy of the blackbox on the adversarial examples in the test is quite low, since the blackbox network is trained on a limited set on a few epochs. However, I still have not been able to fix the tutorial. I tried doubling the holdout, but this barely decreases the accuracy by about 0.05 while taking up double the time as before.

**Comment by npapernot** [comment]
Running the test on the black-box tutorial is hard because it is expensive to do the data augmentation. One way to make the test more representative might be to make the black-box and substitute models more similar in the tutorial itself.

**Comment by npapernot** [comment]
I am closing this given that the problem not addressed yet in this issue is also described in #183. Tests are running more smoothly now.
